scripts/compile-icons.js

```diff
@@ -29,38 +29,41 @@ function pascalCase(x) {
 
 const iconFiles = glob.sync('**/*.svg', { cwd: iconsDir, realpath: true });
 
-iconFiles.forEach(async filePath => {
-  const svgSource = fs.readFileSync(filePath);
+iconFiles.forEach(async (filePath) => {
+  const svgSourceBuffer = fs.readFileSync(filePath);
+  const svgSource = svgSourceBuffer.toString();
 
   try {
-    const viewBoxPosition = svgSource.toString().indexOf('viewBox');
+    const viewBoxPosition = svgSource.indexOf('viewBox');
     if (viewBoxPosition === -1) {
       throw new Error(`${filePath} is missing a 'viewBox' attribute`);
     }
 
-    const jsxSource = await svgr(
+    const jsxSource = await transform(
       svgSource,
       {
         plugins: ['@svgr/plugin-svgo', '@svgr/plugin-jsx'],
         svgoConfig: {
           plugins: [
-            { cleanupIDs: false },
-            { prefixIds: false },
```

Comment:
For reviewers, `prefixIds` is not used anywhere in plugins, it makes no sense to disable it, so it has been removed